src/components/spinner/_spinner.scss

```diff
@@ -47,3 +47,11 @@
     }
   }
 }
+.sr-only {
```

Comment:
what does this do ?

Reply:
it's screen reader only element so need to hide it from visibility. this is common css for screen reader only elements (kms also use it)